open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-server/pom.xml

```diff
@@ -78,6 +85,13 @@
             <artifactId>gremlin-core</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.lucene</groupId>
+            <artifactId>lucene-analyzers-common</artifactId>
+            <version>7.0.0</version>
```

Comment:
Current versions are 8.2.x. However we probably need to be consistent with the rest of egeria (and janus) if possible.  See also #1106 cc: @grahamwallis 

If a new dependency is needed - as with previous comment, please ensure it is added to dependencyManagement and the explicit version removed here. Also I would recommend running 'mvn dependency:tree' in this case, as we have transient dependencies coming in from Janus. (let me know if you need any help)